### DISCUSSION TITLE
Linux on Surface Pro X?

### DISCUSSION THREAD

**Discussion by philliplanos** [linux-surface/linux-surface#931] [General]
It would be a dream to get a punchier, lighter OS onto my surface pro x. I wonder what the possibility of this is in 2022?

**Comment by qzed** [comment]
See https://github.com/linux-surface/surface-pro-x/ (and the issues in that repo, particularly https://github.com/linux-surface/surface-pro-x/issues/1) for the current status. The wiki has some instructions on how to get things running, but it's quite a bit of work and there are still a lot of missing parts.
